### ISSUE TITLE
Broken details sidebar

### ISSUE THREAD

**Issue by XueSheng-GIT** [nextcloud/text#3367]
**Describe the bug**
Opening text does break the details side bar (empty content, sometimes not visible, sometimes visible, but completely empty).

**To Reproduce**
Steps to reproduce the behaviour:
1. Open details sidebar of a markdown (text) file.
2. Click on the text file to open it
3. Close details sidebar
4. Click "3-dots" and "Open sidebar" to open sidebar again

**Expected behaviour**
Sidebar content should be visible if sidebar is opened (step 2 and 4 above).

**Screenshots**
https://user-images.githubusercontent.com/4077683/200179810-fbf58502-3136-41dd-a9c6-67f49c011d52.mp4

**Client details:**
 - OS: Ubuntu 22.04
 - Browser: Firefox
 - Version: 106.0.2
 - Device: Desktop

<details>
<summary>Server details</summary>

**Text app version:** 3.6.0

**Operating system:** Ubuntu 20.04

**Web server:** apache2 (2.4.54)

**Database:** mariadb (Ver 15.1 Distrib 10.3.34-MariaDB)

**PHP version:** 7.4.32

**Nextcloud version:** 25.0.1

</details>

**Comment by mejo-** [comment]
Might this be related to the focus-trap issues we have in Nextcloud 25?

**Comment by juliushaertl** [comment]
Can you maybe tell if your folder/filename has a space or special characters and if it does not happen with just a Readme.md in the root directory?

**Comment by XueSheng-GIT** [comment]
You are right. It does work for "Readme.md" and breaks as soon as a space or special character is added to the Filename!

**Comment by juliushaertl** [comment]
Moving to viewer since the file actions and sidebar handling is happening over there.
